GVRf/Framework/framework/src/main/java/org/gearvrf/GVRContext.java

```diff
@@ -2769,4 +2774,73 @@ public void finalize() throws Throwable {
             super.finalize();
         }
     }
+
+    /**
+     * Show a toast-like message for 3 seconds
+     *
+     * @param message
+     */
+    public void showToast(final String message) {
+        showToast(message, 3f);
+    }
+
+    /**
+     * Show a toast-like message for the specified duration
+     *
+     * @param message
+     * @param duration in seconds
+     */
+    public void showToast(final String message, float duration) {
+        final float quadWidth = 1.2f;
+        final GVRTextViewSceneObject toastSceneObject = new GVRTextViewSceneObject(this, quadWidth, quadWidth / 5,
+                message);
+
+        toastSceneObject.setTextSize(6);
+        toastSceneObject.setTextColor(Color.WHITE);
+        toastSceneObject.setGravity(Gravity.CENTER_HORIZONTAL|Gravity.TOP);
+        toastSceneObject.setBackgroundColor(Color.DKGRAY);
+        toastSceneObject.setRefreshFrequency(GVRTextViewSceneObject.IntervalFrequency.LOW);
```

Comment:
It is possible but it is just an extra overhead. Since the toast text never changes drawing to the canvas once should be enough.